src/Environment.swift

```diff
@@ -148,3 +171,17 @@ public enum Env {
         return ProcessInfo.processInfo.environment[key]
     }
 }
+
+private extension NSColor {
+    convenience init?(withRGBA rgba: String) {
+        let numberRegex = try! NSRegularExpression(pattern: "(\\d+\\.?\\d+)")
+        let nsrgba = rgba as NSString
+        let matches = numberRegex.matches(in: rgba, range: NSRange(location: 0, length: nsrgba.length))
+        let results = matches
+            .map { nsrgba.substring(with: $0.range) }
+            .flatMap { Double($0) }
+            .map { CGFloat($0) }
+        guard results.count == 4 else { return nil }
+        self.init(red: results[0], green: results[1], blue: results[2], alpha: results[3])
+    }
```

Comment:
Ah, that's definitely a bit better. Still horrible, but a better kind of horrible 🙈😄